h2o-docs/src/product/data-science/gam.rst

```diff
@@ -515,6 +519,20 @@ Element at row :math:`m` and column :math:`n` of :math:`penaltyMat` is
    
    penaltyMat_{m,n} = \int_{t_0}^{t_N}{\frac{d^2(I_{m,k}(t))}{d^2t}}{\frac{d^2(I_{n,k}(t))}{d^2t}}dt
 
+*Derivative of M-splines*
+
+The penalty matrix written in terms of the second derivative of M-spline as:
+
+.. math::
+   
+   penaltyMat_{m,n} = \int_{t_0}^{t_N} \frac{dM_{m,k}^2 (t)}{dt^2} \frac{dM_{n,k} (t)}{dt^2}dt
+
+Instead of using the recursive expression, look at the coefficients associated with :math:`M_{m,k}(t)`, take the second derivative, and go from there. This is the procedure to use:
```

Comment:
I think:

penaltyMat_{m,n} = \int_{t_0}^{t_N} \frac{dM_{m,k}^2 (t)}{dt^2} \frac{dM_{n,k} (t)}{dt^2}dt

should be

penaltyMat_{m,n} = \int_{t_0}^{t_N} \frac{d^2M_{m,k} (t)}{dt^2} \frac{d^2M_{n,k} (t)}{dt^2}dt